src/Autocomplete/src/Form/AutocompleteChoiceTypeExtension.php

```diff
@@ -79,6 +84,10 @@ public function finishView(FormView $view, FormInterface $form, array $options):
             $values['min-characters'] = $options['min_characters'];
         }
 
+        if ($options['extra_options']) {
```

Comment:
I have some doubts here, as AutocompleteChoiceTypeExtension support every TextType / ChoiceType (so i'd say 80% of the types used in forms)

Doing so, don't we raise the risk of conflict with userland extensions or other extensions ? 

I did **not** find any other "extra_options" currently in the current Symfony codebase, so it's more a "what if" / "when if" comment. 

This is a problem related to Symfony Form more than this specific PR, but as we extends other types, i'm thinking about having a namespace-ish conventions.

But, as we already define other options without namespace, i would not add confusion with two parralels systems.

So probably "too late" there, but i'd love to have your (and others) opinion on this :)

Reply:
We can add a configuration entry and make this configurable (the name of the property will be passed via a constructor).

Reply:
Yes i do agree.. and as i said it's a broader problem that concern more how the form extensions work than this PR in particular.